### ISSUE TITLE
Default option gives "No files matching the pattern"

### ISSUE THREAD

**Issue by ehannes** [danbruegge/gatsby-plugin-stylelint#6]
Hi!
I just upgrades from version 3.4.1 to 4.0.0 and I got this build error in gatsby:
```
...
success run queries - 2.269s - 10/10 4.41/s                                                                                                                      
                                                                                                                                                                                              
ERROR                                        
                                                                                                                                                                                                                                                            
Error in StylelintBarePlugin                                                                                                                                                                  
                                                                                                                                                                                                                       
ERROR                                           
                                            
Trace: Error: No files matching the pattern "/home/my-user/project/node_modules/@danbruegge/gatsby-plugin-stylelint/src/**/*.{js,jsx}" were found.
    at stylelint.lint.then.catch (/home/my-user/project/node_modules/stylelint-bare-webpack-plugin/lib/index.js:65:21)
```

I added this in `gatsby-config.js` and got it working: `options: { files: ['**/*.{js,jsx}'] }`.

I saw that you changed the default option in version 4.0.0 to
```
files: `${__dirname}/src/**/*.{js,jsx}`
```

The `__dirname` seems to be the plugin folder, not the gatsby folder. Or am I missing some setup?

**Comment by danbruegge** [comment]
Hej.

you are right. This makes sense. I will update as soon as i find time for it. :)

**Comment by danbruegge** [comment]
I've published v4.1.0, let me know if it is working for you.

**Comment by ehannes** [comment]
Thanks @danbruegge, works like a charm!
